views/partials/footer.pug

```diff
@@ -0,0 +1,23 @@
+#footer
+
+    
+  //- CSS Style sheet for footer
+  style.
+    div {
+      text-align: center;
+    }
+
+  div © 2018 Kurt Lewis. Maintained by Kurt Lewis and Sean Kane.
+  div Code licensed under
+
+    //- Link to MIT License, might need to be updated?
+    a.navbar(href="http://opensource.org/licenses/mit-license.html") MIT License
+  div
+    //- Linke to 
+    a.navbar(href="https://github.com/kurtlewis/ceas-ambassadors-website") Updated Project Fork 
```

Comment:
Since this is an original website, we can just change it to say `Project Home` or something similar instead of `Updated Project Fork`